Styleguide/Areas/MyFeature/Pages/Index.cshtml

```diff
@@ -8,7 +7,7 @@
     <title>Style Guide</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <script type="text/javascript" src="https://code.jquery.com/jquery-3.3.1.js" integrity="sha256-2Kok7MbOyxpgUVvAk/HJ2jigOSYS2auK4Pfzbm7uH60="crossorigin="anonymous"></script>
```

Comment:
can we quit using jquery ? it seems there's no rocket science here and we can replace it with vanilla js